contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/gobblin/build.gradle

```diff
@@ -0,0 +1,48 @@
+apply plugin: 'com.github.johnrengelman.shadow'
+
+repositories {
+    jcenter()
+    maven {
+        // For: "org.pentaho:pentaho-aggdesigner-algorithm:5.1.3-jhyde"
+        // required by: "gobblin-runtime:0.10.0 > org.apache.hive:hive-exec:1.0.1"
+        url "http://repo.spring.io/plugins-release"
+    }
+}
+
+configurations {
+    gobblin.exclude group: 'com.google.guava'
+    resolvedIncorrectly
```

Comment:
This is a new configuration.
Theoretically we can put everything in the same configuration, but for some reason gradle won't resolve guava dependencies correctly. Actually it displays the correct version but will download a different version during build. Therefore I'm adding another configuration just for guava.